### PR TITLE
CEDS-1825 - movements controller specs

### PR DESCRIPTION
**Pull request by SteveSugden**

### CODE REVIEW

File: test/controllers/movements/SummaryControllerSpec.scala [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+    "return 200 (OK)" when {
+
+      "GET displayPage is invoked without data in cache" in {
+
+        givenTheCacheIsEmpty()
+
+        val result = controller().displayPage()(getRequest)
+
+        status(result) mustBe OK
```
Comment (ed0906) [inline test/controllers/movements/SummaryControllerSpec.scala]: Can we assert on what view was sent in the response please :)

File: test/repository/MockCache.scala [hunk 2]
```diff
@@ -50,6 +50,9 @@ trait MockCache extends MockitoSugar with BeforeAndAfterEach {
   protected def givenTheCacheIsEmpty(): Unit =
     given(cache.findByPid(any())).willReturn(Future.successful(None))
 
+  protected def givenTheCacheIsCleared(): Unit =
```
Comment (ed0906) [inline test/repository/MockCache.scala]: Dont think we need a method for this, we can just stick it in the `beforeEach`

Comment (SteveSugden) [reply test/repository/MockCache.scala]: Done

File: test/controllers/movements/SummaryControllerSpec.scala (hunk 1 above)
Comment (SteveSugden) [reply test/controllers/movements/SummaryControllerSpec.scala]: Done

**Pull request merged**